### DISCUSSION TITLE
Camera position and rotation

### DISCUSSION THREAD

**Discussion by shifugate** [CariusLars/ar_flutter_plugin#79] [Ideas]
Hello.

Sorry for my english.
I did need get camera position and rotation in worldspace and add to plugin in iOS and Android a less code to do it.
I put the code here, maybe help.

Best regards.

[AndroidARView.txt](https://github.com/CariusLars/ar_flutter_plugin/files/7870131/AndroidARView.txt)
[IosARView.txt](https://github.com/CariusLars/ar_flutter_plugin/files/7870132/IosARView.txt)

**Comment by CariusLars** [comment]
Hey @shifugate ,

I actually implemented this in a branch that has then gone stale (https://github.com/CariusLars/ar_flutter_plugin/tree/feature/gps-objects), you can see that the [ARSessionManager](https://github.com/CariusLars/ar_flutter_plugin/blob/feature/gps-objects/lib/managers/ar_session_manager.dart) has a `getCameraPose` function that is implemented in the AndroidARView and the IosARView accordingly.
If you want you could copy the functionality over and create a pull request, I can then merge it into main :)

Cheers,
Lars
